### DISCUSSION TITLE
Do scraped web data ever get deleted?

### DISCUSSION THREAD

**Discussion by Kanishk-Kumar** [open-webui/open-webui#2875] [Q&A]
Hi, I'm not seeing it getting deleted anywhere:
https://github.com/open-webui/open-webui/blob/main/src/lib/apis/rag/index.ts
https://github.com/open-webui/open-webui/blob/main/backend/apps/rag/main.py
What if I'm querying about thousands of #URLs everyday? Wouldn't it end up making the vector DB slow?

**Comment by Classic298** [comment]
Interesting, so web scraped data doesnt get deleted at all even if you delete the chat?
